src/Components/FilterButton/FilterButton.module.css

```diff
@@ -0,0 +1,48 @@
+.filterCard {
+  height: 150px;
+  box-shadow: 0 0 6px var(--color-7);
+  font-size: 0.9rem;
+  padding: 1rem 2rem;
+  margin: 0.5rem;
+  width: 200px;
+}
+
+.title {
+  margin-top: 0.5rem;
+  font-weight: bold;
+  margin-bottom: 1rem;
+}
+
+.section {
+  margin-bottom: 1rem;
+}
+
+.section input {
+  margin-right: 0.3rem;
+}
+
+@media screen and (max-width: 720px) {
+  .filterCard {
+    height: 150px;

```

Comment:
remove it from here, already defined in the common style

Reply:
Done. Refactored CSS to reduce redundancy.